optax/_src/loss_test.py

```diff
@@ -416,5 +416,56 @@ def test_repeat_with_one_to_one_alignment(self):
           jnp.array(expected_loss), per_seq_loss[n], rtol=self._rtol)
 
 
+class KLDivergenceTest(parameterized.TestCase):
+
+  def setUp(self):
+    super().setUp()
+    self.log_ps = np.array([[-2.9957, -3.5066, -3.9120, -1.2040, -0.6931, -2.3026],
+                            [-1.6094, -1.6094, -1.6094, -2.3026, -1.8971, -1.8971]])
+    self.qs = np.array([[0.2, 0.2, 0.2, 0.1, 0.15, 0.15],
+                       [0.05, 0.03, 0.02, 0.3, 0.5, 0.1]])
+    # computed kullback-leibler divergence of P from Q.
+    self.exp = np.array([0.8875625, 0.7187435584901326])
+
+  @chex.all_variants
+  def test_scalar(self):
+    np.testing.assert_allclose(
+        self.variant(loss.kl_div_loss)(self.log_ps[0], self.qs[0]),
```

Comment:
This (and the other function names) should be updated for the test to pass.

Reply:
done!